src/lib/types.ts

```diff
@@ -39,6 +46,7 @@ export interface IPost {
   modifiedAt?: string;
   postType: PostTypeEnum;
   preview?: string;
+  id?: number;
```

Comment:
it should be required.
do we have any cases when it's optional?

Reply:
Yep, in `fetchImportantPosts` thunk, we don't use the `id` field.

[fetchImportantPosts](https://github.com/ita-social-projects/dokazovi-fe/blob/4530fd690bfe3d48a9269323171f14b4768562ed/src/modules/main/store/mainSlice.ts#L135-L159)